### ISSUE TITLE
Look for test files recursively

### ISSUE THREAD

**Issue by itstehkman** [hillstreetlabs/espresso#43]
https://github.com/hillstreetlabs/espresso/blob/81e74adadacce3815879b377e50979c44c4ce28e/src/espresso.js#L67

This would be a nice feature and is a quick fix. You may also want to consider allowing several test paths from the command line.

**Comment by pfletcherhill** [comment]
Great idea. Want to make a PR? ;)

**Comment by itstehkman** [comment]
@pfletcherhill yes i'd love to! I'm trying to use espresso in [zos-cli](https://github.com/zeppelinos/zos-cli/pull/241) to speed up some tests. Solidity testing is 🐌

**Comment by pfletcherhill** [comment]
@itstehkman that would be awesome! @mertcelebi recently pushed [some updates](https://github.com/hillstreetlabs/espresso/pull/44) after using espresso with [kimono](https://github.com/hillstreetlabs/kimono). But we admittedly have had trouble finding time to make significant changes. Would really appreciate your help if you're already digging into the code.

**Comment by itstehkman** [comment]
I don't have much time either but if I see an opportunity to make a PR, I'll go ahead and do that :D
